src/SprykerEco/Zed/Optivo/OptivoConfig.php

```diff
@@ -26,24 +34,124 @@ public function getRequestTimeout()
     /**
      * @return string
      */
-    public function getTokenOperationSubscribe()
+    public function getOrderListAuthCode(): string
```

Comment:
Would be nice to rename method to same as constant name.

Reply:
done